common/djangoapps/util/views.py

```diff
@@ -222,6 +223,34 @@ def get_group(self, name):
         return None
 
 
+def _get_zendesk_custom_field_context(request):
+    """
+    Construct a dictionary of data that can be stored in Zendesk custom fields.
+    """
+    context = {}
+    course_id = request.POST.get("course_id")
+    if course_id:
+        context["course_id"] = course_id
+        if request.user.is_authenticated():
```

Comment:
#Nitpick I feel like you can flatten these nested if statements. It would be easier to read for me